src/java.desktop/share/native/common/font/sunfontids.h

```diff
@@ -82,10 +82,7 @@ typedef struct FontManagerNativeIDs {
       glyphImages, glyphListUsePos, glyphListPos, lcdRGBOrder, lcdSubPixPos;
 } FontManagerNativeIDs;
 
-/* Note: we share variable in the context of fontmanager lib
-   but we need access method to use it from separate rasterizer lib */
```

Comment:
That comment had nothing to do with the rasterizer OpenJDK project which wasn't even about font rasterisation. I think this refers to the closed source font rasteriser now removed. Anyway I don't think the comment is important enough to insist on keeping it.

Reply:
good to know, thanks!